config/config.go

```diff
@@ -773,10 +773,10 @@ func (c *ScrapeConfig) Validate(globalConfig GlobalConfig) error {
 	}
 
 	switch globalConfig.MetricNameValidationScheme {
-	case "", LegacyValidationConfig:
-	case UTF8ValidationConfig:
+	case LegacyValidationConfig:
+	case "", UTF8ValidationConfig:
 		if model.NameValidationScheme != model.UTF8Validation {
-			return fmt.Errorf("utf8 name validation requested but feature not enabled via --enable-feature=utf8-names")
+			return fmt.Errorf("utf8 name validation requested but validation mode not UTF8 in common/model")
```

Comment:
Saying `common/model` may not be a user friend error message.
```suggestion
			return fmt.Errorf("utf8 name validation requested but validation mode is not set to UTF8")
```